content/faq.md

```diff
@@ -55,51 +76,82 @@ It’s also worth noting that, while there are almost 8B people on Earth, there
+It’s true that Nock, without jets, is slow. With the ability to call out to a
+native function, it’s not so bad. This is done by our runtime,
+[Vere](@/docs/glossary/vere.md), a Nock interpreter written in C.
 
-One, you own your Urbit ID with a key that you never have to give your host. Your host only gets an operating key, and you can always revoke that key remotely. So, you might lose your data but it’s up to you to take good care of your identity.
+### Can I host my Urbit with someone else? {#why-hosting}
 
-Two, your Urbit runs inside a VM. There’s a very clear line over which a host should never step: your event log. This is not unlike the way that cloud hosting works today. Netflix is hosted on AWS — one of their biggest competitors. Luckily, the AWS terms of service protects Netflix from a legal standpoint.
+Yes, [Tlon](@/blog/hosting.md) and [a few others](@/blog/providers.md) began [accepting
+signups](https://tlon.io) for our hosting service in 2020. Under this
+arrangement, Tlon does all the work of setting up cloud infrastructure and
+getting your planet up and running, making getting started with Urbit as simple
+as signing up for an ordinary online service.
 
-At the end of the day, if you’re really worried about security, it’s quite easy to host Urbit yourself. This means that people who care a lot about privacy can still get the same usability as those who want the convenience of hosting.
+Should you one day wish to run your Urbit yourself, any hosting provider ought
+to offer you the ability to download your [ship](@/docs/glossary/ship.md)'s [pier](@/docs/glossary/pier.md)
+to your own device and do so. This is a very low frction process thanks to
+Urbit's deterministic design.
 
 ## Urbit ID
 
 ### What is Urbit ID? {#what-is-urbit-id}
 
-Urbit ID is a general-purpose PKI ("public key infrastructure") that Urbit uses as an identity system. This system is implemented as a suite of smart contracts on the Ethereum blockchain, and it determines which Ethereum addresses own which Urbit planets, stars, or galaxies. In Urbit OS, a single identity is called a "ship", whereas in Urbit ID a single identity is just an "identity."
+Urbit ID is a general-purpose decentralized PKI ("public key infrastructure")
+that Urbit uses as an identity system. This system is implemented as a suite of
+smart contracts on the Ethereum blockchain, and it determines which Ethereum
+addresses own which Urbit [planets](@/docs/glossary/planet.md),
+[stars](@/docs/glossary/star.md), or [galaxies](@/docs/glossary/galaxy.md). In
+[Urbit OS](#what-is-arvo), a single identity is called a
+"[ship](@/docs/glossary/ship.md)", whereas in Urbit ID a single identity is just
+an "identity."
```

Comment:
This last bit is confusing. I also feel like the bulk of this explanation should live under the Azimuth section. Azimuth is the PKI, Azimuth is the contracts on Ethereum. Urbit ID is just an easy way for us to talk about Azimuth in the context of Urbit, right?

Reply:
Agreed. Also Azimuth identities are points even though there has been an effort to eradicate that language, it exists in the code so seems kind of silly.

Reply:
https://github.com/urbit/urbit.org/pull/538/commits/4e2546884b181f42efd8ff460b05d78f42b424a4